src/components/screens/DocsScreen/FrameworkSupportTable.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+
+export function frameworkSupportsFeature(framework, { supported, unsupported }) {
+  return (
+    (supported && supported.includes(framework)) ||
+    (unsupported && !unsupported.includes(framework))
+  );
+}
+
+const frameworkTitle = (framework) =>
+  framework === 'html' ? 'HTML' : framework[0].toUpperCase() + framework.slice(1);
+
+export const FrameworkSupportTable = ({ currentFramework, frameworks, features }) => {
+  return (
+    <table>
+      <thead>
+        <tr>
+          {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
+          <th />
+          {frameworks.map((framework) => (
+            <th>
+              {/* TODO: this should be a link */}
+              {frameworkTitle(framework)}
```

Comment:
Not sure where we should link -- thoughts @shilman?

Reply:
Ideally to the package on `next` or `master` OR relative link to the docs site on that framework if we end up doing framework-specific intros for each framework in the future.

Reply:
Ok!